data_pipeline/sentiment/dags/crypto/el_airflow_crypto_avalanche.py

```diff
@@ -0,0 +1,56 @@
+# imports important for Airflow
+import pendulum
+from airflow.decorators import dag, task
+import logging
+import time
+from newsapi import NewsApiClient
+from airflow.providers.mongo.hooks.mongo import MongoHook
+from datetime import datetime, timedelta
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+@dag(
+    schedule_interval=None,
+    start_date=pendulum.datetime(2024, 1, 1, tz="UTC"),
+    catchup=False,
+    tags=[f'Loading raw sport avalanche data into Airflow'],
+)
+
+def el_crypto_avalanche_data_into_mongodb():
```

Comment:
What is "el"? I don't think this is the best name for this function. Try to come up with a function name without the "el" in it. 

"crypto avalanche"? Hehehe, nah